boa_engine/src/vm/opcode/define/mod.rs

```diff
@@ -66,57 +66,26 @@ impl Operation for DefInitVar {
     }
 }
 
-/// `DefLet` implements the Opcode Operation for `Opcode::DefLet`
+/// `InitializeLexical` implements the Opcode Operation for `Opcode::InitializeLexical`
 ///
 /// Operation:
-///  - Declare `let` type variable.
+///  - Initialize a lexical binding.
 #[derive(Debug, Clone, Copy)]
-pub(crate) struct DefLet;
+pub(crate) struct InitializeLexical;
 
-impl Operation for DefLet {
-    const NAME: &'static str = "DefLet";
-    const INSTRUCTION: &'static str = "INST - DefLet";
+impl Operation for InitializeLexical {
+    const NAME: &'static str = "InitializeLexical";
+    const INSTRUCTION: &'static str = "INST - InitializeLexical";
 
     fn execute(context: &mut Context<'_>) -> JsResult<CompletionType> {
         let index = context.vm.read::<u32>();
+        let value = context.vm.pop();
         let binding_locator = context.vm.frame().code_block.bindings[index as usize];
         context.vm.environments.put_declarative_value(
```

Comment:
Hmm, maybe this should be renamed to `put_lexical_value`?

Reply:
Do you mean to opcode as `PutLexicalValue`?

Reply:
No, I mean the `put_declarative_value` -> `put_lexical_value`. Just thought it was a better name for the function, since it's used inside `InitializeLexical`